packages/core/src/StateNode.ts

```diff
@@ -24,7 +24,6 @@ import type {
   InvokeDefinition,
```

Comment:
idea: could we add a dev-only warning to the `machine.transition` that would warn if somebody tries to send `event.type === ''`? Let's require a non-empty string there

Reply:
Yes, good idea

Reply:
I've forbidden these